doc/lsst.pipe.base/command-line-task-config-howto.rst

```diff
@@ -0,0 +1,141 @@
+.. _command-line-task-config-howto:
+
+##############################
+Configuring command-line tasks
+##############################
+
+:ref:`Command-line tasks <using-command-line-tasks>` are highly configurable.
+This page describes how to review configurations with :option:`--show config <--show>`,  and change configurations on the command line with :option:`--config` or :option:`--configfile`.
+
+One special category of configuration is subtask retargeting.
+Retargeting uses concepts discussed on this page, but with additional considerations described in :doc:`command-line-task-retargeting-howto`.
+
+.. _command-line-task-config-howto-show:
+
+How to show a command-line task's current configuration
+=======================================================
+
+The :option:`--show` argument allows you to review the current configuration details for command-line tasks. 
+
+.. _command-line-task-config-howto-show-all:
+
+Viewing all configurations
+--------------------------
+
+Use :option:`--show config <--show>` to see all current configurations for a command-line task.
+For example:
+
+.. code-block:: bash
+
+   processCcd.py REPOPATH --output output --show config
+
+Note that both the input and output Butler data repositories repository must be set to show camera-specific task configuration defaults.
+Instead of an :option:`--output` argument you can specify a :option:`--rerun`, see :doc:`command-line-task-data-repo-howto`.
+
+.. _command-line-task-config-howto-show-subset:
+
+Viewing a subset of configurations
+----------------------------------
+
+To filter the :option:`--show config <--show>` output, include a search term with wildcard (``*``) characters.
+For example, this will show any configuration that includes the string ``calibration``:
+
```

Comment:
When we did `--show DATA` instead of `--show config`, you pointed out in a friendly box that this would put the task into dry-run mode. Is this the case for all tasks run with `--show`, or just ones where you feed `--show` a data repository? (The same question applies to the later instance of `--show tasks`.)